<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditNodeOrder</class>
 <widget class="QDialog" name="EditNodeOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>-1</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="_2">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rBtnCash">
          <property name="text">
           <string>Cash</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnMonthly">
          <property name="text">
           <string>Monthly</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnPending">
          <property name="text">
           <string>Pending</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="DateEdit" name="dateEdit">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
          <property name="timeSpec">
           <enum>Qt::TimeSpec::LocalTime</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,2,1,1,2,1,1,2,1,2,1">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="labelParty">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Party</string>
          </property>
          <property name="buddy">
           <cstring>comboParty</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ComboBox" name="comboParty">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnInsertParty">
          <property name="text">
           <string>InsertParty</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="DoubleSpinBox" name="dSpinInitialTotal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelInitialTotal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>InitialTotal</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DoubleSpinBox" name="dSpinFinalTotal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <italic>false</italic>
            <bold>true</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDiscount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Discount</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
          </property>
          <property name="buddy">
           <cstring>dSpinDiscount</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DoubleSpinBox" name="dSpinDiscount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ComboBox" name="comboEmployee">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEmployee">
          <property name="text">
           <string>Employee</string>
          </property>
          <property name="buddy">
           <cstring>comboEmployee</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" rowspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkBoxRefund">
          <property name="text">
           <string>Refund</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkBoxBranch">
          <property name="text">
           <string>&amp;Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEdit" name="lineEditOrder">
          <property name="placeholderText">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnInsertOrder">
          <property name="text">
           <string>InsertOrder/todo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnPrint">
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableViewOrder"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="pBtnDeleteOrder">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnSaveOrder">
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnPostOrder">
       <property name="text">
        <string>&amp;Post</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">widget/doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widget/combobox.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widget/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">widget/dateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableViewOrder</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
